webapp/frontend/app/controllers/user/login.js

```diff
@@ -29,24 +29,30 @@ actions : {
       crossDomain: true,
       method: 'GET',
       dataType: 'json',
-      data: headers,
       xhrFields: {withCredentials: true},
       success: function(){
           _this.set("isProcessing", false);
           _this.set("loginFailed", false);
           _this.set("serverError", false);
 
+          _this.store.createRecord('user', {
+              token: token,
+              id: 1,
+          });
+
           _this.transitionToRoute('user.clusters');
```

Comment:
Please change name `clusters`, we don't use it any more. Use the name `lambda-instances` instead.